.travis.yml

```diff
@@ -49,10 +60,8 @@ services:
  - memcache
  - redis-server
 
-before_install:
-  - sudo apt-get install -y axel cpanminus libtest-base-perl libtext-diff-perl liburi-perl libwww-perl libtest-longstring-perl liblist-moreutils-perl libgd-dev
-
 install:
+  - mkdir ~/bin && ln -s $(which ccache) ~/bin/$CC && export PATH=$HOME/bin:$PATH
```

Comment:
@chipitsine I reread the manpage of ccache and I failed to see any propounding reason of using the masquerading approach other than convenience when ccache'ing all the compilation in the current session. Well, given that Travis CI is using a VM approach and invasive settings are common practice, so maybe it also makes things more convenient if not really elegant.